tests/saving_data_test.py

```diff
@@ -1,20 +1,16 @@
-"""
-Test the saving data capability
-"""
+"""Test the saving data capability."""
```

Comment:
There's a few of these. I have a pydocstyle pre-commit hook, so have to add this kind of thing.